apps/ledger-live-mobile/src/families/elrond/components/Flows/Delegate/components/PickAmount/PickAmount.tsx

```diff
@@ -47,7 +46,7 @@ const PickAmount = (props: PickAmountPropsType) => {
 
   const getMaxSpendable = useCallback(() => {
     const fetchMaxSpendable = async () => {
-      const amount = await estimateMaxSpendable({
+      const amount = await bridge.estimateMaxSpendable({
```

Comment:
missing `bridge` dependency for useCallback

Reply:
FYI: You can catch those errors quickly with `pnpm lint` in the root of the repo